sacred/observers/mongo.py

```diff
@@ -94,6 +95,19 @@ def __init__(self, runs_collection,
         self.run_entry = None
         self.priority = priority
 
+    def find_queued(self, query):
```

Comment:
I think this should be called `acquire_queued` to better reflect the fact that it updates the entry.

Reply:
Agreed.